01_example.ipynb

```diff
@@ -42,8 +42,8 @@
     "#| export\n",
```

Comment:
<p>Why have you changed the number of data points?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LauraRichter/nbdev-demo/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Can you please add a doctoring</p><p> </p>